src/leaflike/core.clj

```diff
@@ -1,24 +1,24 @@
 (ns leaflike.core
-  (require [org.httpkit.server :as httpkit]))
+  (require [leaflike.server :refer [start-server stop-server]]
```

Comment:
Does this work? I thought you needed to use the `:require` form here.

Reply:
From https://clojuredocs.org/clojure.core/require:
Use :require in the ns macro in preference to calling this directly.
Use the function `require` only in the REPL.